Lib/gzip.py

```diff
@@ -10,6 +10,7 @@
 import builtins
 import io
 import _compression
+import weakref
```

Comment:
the ordering of imports here is painful to me, but resisted bringing to PEP8 because this fix needs backporting.

Reply:
Please reformat these imports for PEP 8 :-) I will handle merge conflicts on backports if needed.